package.json

```diff
@@ -11,6 +11,7 @@
     "travis-deps": "pip install -t lib -r requirements.travis.txt --upgrade",
     "dev-deps": "python -m pip install --no-deps -r requirements.dev.txt",
     "test": "python -m unittest discover -p '*_test.py'  -b",
+    "emulator" : "gcloud beta emulators datastore start --host-port=:15606 --no-store-on-disk --consistency=1.0",
```

Comment:
Some tests were failing because the default consistency of the emulator is 0.9. As mentioned [here](https://cloud.google.com/sdk/gcloud/reference/beta/emulators/datastore/start), 

> --consistency=CONSISTENCY; default=0.9
Fraction of eventually consistent operations that should succeed immediately. Setting to 1.0 can be useful for unit tests, but may mask incorrect assumptions about non-ancestor queries which are eventually consistent.

Setting the consistency to 1.0 makes the test cases run successfully.